Website/src/rendering/src/assets/css/components/NonSitecore/InformationPageHero.css

```diff
@@ -109,7 +109,8 @@
   }
 
   &.premium,
-  &.featured {
+  &.featured,
+  &.gold {
```

Comment:
Please add a Storybook InformationPageHero variant for each qualificative. Premium, Featured, Gold, Silver, and Platinum.